atbuild/src/main.swift

```diff
@@ -76,7 +73,7 @@ func runtask(taskName: String) {
 }
 
 //choose which task to run
-var run = false
+var task : String? = nil
```

Comment:
I'd suggest `taskName` to clarify that it's not actually an instance of the task.